src/Standards/PSR12/Sniffs/Methods/NullableWhitespaceSniff.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Verifies that nullable typehints are lacking superfluous whitespace, e.g. ?int
+ *
+ * @author    Greg Sherwood <gsherwood@squiz.net>
+ * @copyright 2006-2015 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Standards\PSR12\Sniffs\Methods;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class NullableWhitespaceSniff implements Sniff
+{
+
+
+    /**
+     * Returns an array of tokens this test wants to listen for.
+     *
+     * @return array
+     */
+    public function register()
+    {
+        return [T_NULLABLE];
+
+    }//end register()
+
+
+    /**
+     * Processes this test, when one of its tokens is encountered.
+     *
+     * @param \PHP_CodeSniffer\Files\File $phpcsFile The file being scanned.
+     * @param int                         $stackPtr  The position of the current token in the
+     *                                               stack passed in $tokens.
+     *
+     * @return void
+     */
+    public function process(File $phpcsFile, $stackPtr)
+    {
+        $tokens = $phpcsFile->getTokens();
+        if (isset($tokens[($stackPtr + 1)]) === true
```

Comment:
Looks like you are only checking (and fixing) the first token after the `?`.

What about if there is `whitespace new-line whitespace` ? or what about if there is a comment or PHPCS annotation in between ?
```php
function ( ?
    string $varA,
    ? /* comment about why it is nullable */ array $varB,
    ? // phpcs:ignore Standard.Cat.Sniff -- for reasons
    int $varC
) {}
```
I'm not saying that sort of code is clean by any standard, but IMHO this sniff should not break on it.